aspnetcore/aspnetcore-2.1.md

```diff
@@ -0,0 +1,188 @@
+
+Applications that do **not** include authentication can apply the scaffolder to add the RCL Identity package. You have the option of selecting Identity code to be generated.
+
+For more information, see [Scaffold Identity in ASP.NET Core projects](xref:security/authentication/scaffold-identity)
+
+## HTTPS
+
+With the increased focus on security and privacy, enabling HTTPS for web apps is more important than ever before. HTTPS enforcement is becoming increasingly strict on the web. Sites that don’t use HTTPS are considered insecure. Browsers are starting to enforce that many new and existing web features must only be used from a secure context (Chromium, Mozilla). GDPR requires the use of HTTPS to protect user privacy. While using HTTPS in production is critical, using HTTPS during development can also help prevent related issues before deployment, like insecure links. For more information see:
+
+* [HTTP Strict Transport Security Protocol (HSTS)](xref:security/enforcing-ssl#http-strict-transport-security-protocol-hsts)
+* [Require HTTPS](xref:security/enforcing-ssl#require-https)
+
+### On by default
+
+To facilitate secure website development, HTTPS in ASP.NET Core 2.1 is now enabled by default. Starting in 2.1, Kestrel will listen on `https://localhost:5001` when a local development certificate is present. A certificate will be created:
+
+* When the .NET Core SDK is installed.  Run `dotnet dev-certs https --trust`.
+* By manually set up using the new `dev-certs` tool.
+
+The project templates have updated to:
+
+* [Require HTTPS](xref:security/enforcing-ssl#require-https) by default.
+* Include [HTTPS redirection](xref:security/enforcing-ssl#require-https) and [HTTP Strict Transport Security Protocol (HSTS)](xref:security/enforcing-ssl#http-strict-transport-security-protocol-hsts).
+
+### HTTPS redirection and enforcement
+
+Web apps typically need to listen on both HTTP and HTTPS, but then redirect all HTTP traffic to HTTPS. ASP.NET Core 2.0 has URL rewrite middleware that can be used for this purpose, but there are challenges to configure this redirection. In 2.1, specialized HTTPS redirection middleware that intelligently redirects based on the presence of configuration or bound server ports has been introduced.
+
+Use of HTTPS can be further enforced using [HTTP Strict Transport Security Protocol (HSTS)](xref:security/enforcing-ssl#http-strict-transport-security-protocol-hsts). HSTS  instructs browsers to always access the site via HTTPS. ASP.NET Core 2.1 adds HSTS middleware that supports options for max age, subdomains, and the HSTS preload list.
+
+### Configuration for production
+
+In production, HTTPS must be explicitly configured. In 2.1, default configuration schema for configuring HTTPS for Kestrel has been added. Applications can be configured to use:
+
+* Multiple endpoints including the URLs.
```

Comment:
Add link to docs on how to do this

Reply:
@guardrex  or @danroth27  can you give me a link on `Multiple endpoints including the URLs.`

Reply:
@guardrex  can you add that? I'm breaking for lunch now.